lib/mdl/rules.rb

```diff
@@ -176,7 +176,7 @@
 rule "MD013", "Line length" do
   tags :line_length
   aliases 'line-length'
-  params :line_length => 80, :code_blocks => true, :tables => true
+  params :line_length => 80, :code_blocks => true, :tables => true, :hard_wrap => true
```

Comment:
how about `:style => :wrapped` and `:style => :not_wrapped`

Reply:
Can do… my instinct was to go with a boolean since it’s a binary thing. I tend to avoid enum style parameters when it’s a binary, but I agree it is clearer and it’s just an instinct.

Reply:
How would you suggest I handle cases where `:style` is set to some other symbol? I’m replacing my if/else with an if/elsif to make it clear that only those two cases are handled.

Reply:
Interestingly I just spoke to a colleague who likes to wrap at the end of each sentence, so now I can see that perhaps other styles might emerge in the future.

Reply:
There's already an issue out for that, #315 - it's why I wanted to do `style`. :)

Reply:
Ah, nice… I hadn’t seen that issue. 😃

Reply:
@jaymzh as per the conversation on https://github.com/DavidAnson/markdownlint/issues/298, perhaps this change should be introduced at the same time? Would you like me to see if I can include it in this PR?

Reply:
sentence-based? Sure!